apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx

```diff
@@ -65,8 +63,7 @@ const compileGridData = (market: Market_market) => {
       label: isLiquidityMonitoringAuction
         ? t('Est auction end')
         : t('Auction end'),
-      value: getDateTimeFormat().format(new Date(market.data.auctionEnd)),
-      isEstimate: isLiquidityMonitoringAuction ? true : false,
```

Comment:
question (blocking?): not sure if this was a change in requirements here? are we now displaying all market types prefixed with a `~`?

Reply:
@mattrussell36 removed it here https://github.com/vegaprotocol/frontend-monorepo/pull/1133/commits/57bcec59d8e02c6b71ca3ad6bde3a0369113b525  was this in the design? perhaps we should check with @campbellssource

Reply:
Good catch. I've amended it to include the tilde when in its a liq monitoring auction